packages/graphql/tests/performance/graphql/batch-create.graphql

```diff
@@ -1,4 +1,4 @@
-mutation BatchCreate_skip {
+mutation BatchCreate {
```

Comment:
Isn't this duplicate of the other `batch-create.graphql`?

Reply:
Yes, we moved to the folder /batch-create, I thought I had remove it 👍